Gemfile

```diff
@@ -6,13 +6,7 @@ source "https://rubygems.org"
 gemspec
```

Comment:
Since we have this line, I _think_ we don't need to redeclare the gemspec dependencies in the Gemfile?

Reply:
Yeah, you're right